src/libs/ReportUtils.js

```diff
@@ -937,6 +937,32 @@ function getDisplayNamesWithTooltips(participants, isMultipleParticipantReport)
     });
 }
 
+/**
+ * We get the amount, currency and comment money request value from the action.originalMessage.
+ * But for the send money action, the above value is put in the IOUDetails object.
+ *
+ * @param {Object} reportAction
+ * @param {Number} reportAction.amount
+ * @param {String} reportAction.currency
+ * @param {String} reportAction.comment
+ * @param {Object} reportAction.IOUDetails
```

Comment:
to make sure its clear its not there every time
```suggestion
 * @param {Object} [reportAction.IOUDetails]
```